unattended_installer/install_functions/opendistro/checks.sh

```diff
@@ -155,42 +161,83 @@ function checkArguments() {
 
 }
 
+function checkTools() {
+
+    # -------------- Check tools required to run the script (awk, sed, etc.) -----------------------------------------
+
+    toolList=(  "awk" "cat" "chown" "cp" "curl" "echo" "export"
+                "free" "grep" "kill" "mkdir" "mv" "rm" "sed"
+                "sudo" "tar" "touch" "uname")
+
+    missingtoolsList=()
+    for command in "${toolList[@]}"
+    do
+        if [ -z "$(command -v ${command})" ]; then
+            missingtoolsList+="${command}, "
+        fi
+    done
+
+    if [ -n "${missingtoolsList}" ]; then
+
+        logger "---------------------------------- Missing tool -----------------------------------"
+        logger "Missing tool report: ${missingtoolsList} are not installed. All this command's are necessary for the correct use of this tool."
+        exit 1
+
+    fi
+
+}
+
 function checkHealth() {
 
     checkSpecs
-    if [ -n "${elasticsearch}" ]; then
-        if [ "${cores}" -lt 2 ] || [ "${ram_gb}" -lt 3700 ]; then
-            logger -e "Your system does not meet the recommended minimum hardware requirements of 4Gb of RAM and 2 CPU cores. If you want to proceed with the installation use the -i option to ignore these requirements."
-            exit 1
-        else
-            logger "Check recommended minimum hardware requirements for Elasticsearch done."
-        fi
+    if [ -z "${cores}" ]; then
+        logger -w "The script needs to parse the file '${coresFile}' to check the minimum required hardware of CPU cores."
```

Comment:
If you plan to exit from the script run, this message is not a warning but an error. Only errors should be able to exit from a run.

Reply:
Done